app/src/main/res/layout/detail_fragment.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="270dp"
+        android:id="@+id/videoLayout">
+
+        <ImageView
+            android:id="@+id/imgvMoviePoster"
```

Comment:
I think these two ids are reversed. This is actually the backdrop and the one below is the poster. Right?